h5py/h5s.pyx

```diff
@@ -47,7 +47,7 @@ UNLIMITED = H5S_UNLIMITED
 NO_CLASS = H5S_NO_CLASS
 SCALAR   = H5S_SCALAR
 SIMPLE   = H5S_SIMPLE
-_NULL = H5S_NULL
+globals()["NULL"] = H5S_NULL  # "NULL" is reserved in Cython
```

Comment:
This is intentional... Cython doesn't allow "NULL = H5S_NULL" since "NULL" is a reserved word.  We used to patch this up in __init__.py.